src/test/java/com/bigchaindb/api/TransactionCreateApiTest.java

```diff
@@ -289,6 +291,7 @@ public void testBuildTransaction() {
             assertTrue(transaction.getVersion().equals("2.0"));
             assertTrue(transaction.getAsset().getData() != null);
             assertTrue(transaction.getSigned());
+            assertEquals(transaction.getOperation(), "CREATE");
```

Comment:
add ability in tests to enforce `CREATE` operation type

Reply:
good find. accepted